packages/sitecore-jss/styles.d.ts

```diff
@@ -0,0 +1 @@
+export * from './types/layout/styles/index';
```

Comment:
Is there any reason, or benefit of having additional /styles submodule? Since it's related to _layout_

Reply:
can't think of any other than reducing the amount of files in the folder but since the number is not too high we can remove /styles; on the other hand - if we decide to keep it - maybe it would make sense to move content-styles under /styles too;

Reply:
I would prefer to export this functionality from _/layout_ submodule, I think it should be still fine.
We will remove _feaas_ submodule in JSS 22